pkg/actions/addon/tasks.go

```diff
@@ -58,6 +60,9 @@ func (t *createAddonTask) Do(errorCh chan error) error {
 	}
 
 	for _, a := range t.addons {
+		if t.forceAll {
+			a.Force = true
+		}
```

Comment:
Is there an opportunity to set this to not be `true`? I guess I am wondering why we have it as a configurable var, when nothing seems to allow that value to be changed. The call in `pkg/ctl/create/cluster.go` simply sets it to `true`.

Reply:
yeah I did think about hard coding it straight in, but its not really clear that the tasks is only currently being used by the create cluster flow, it could be used elsewhere in the future (thinking apply as an option), so having it as something configured just makes it clear whats happening.